pkgs/applications/editors/rcptt/default.nix

```diff
@@ -0,0 +1,85 @@
+# Intende useage
+# Build the elexis-congo app via mvn -V clean verify -f elexis-congo/pom.xml
+# nix-build Elexis_Congo.nix
+# start one of the binaries under result/bin/
+#
+{ lib, pkgs, stdenv, bundlerEnv, ruby, makeWrapper, makeDesktopItem
+, gnome, git, gtk3, jdk11, fontconfig, freetype, glib
+, gsettings-desktop-schemas, webkitgtk, docutils, perl, xorg, zlib
+, libXrender, libsecret, unzip, rsync, fetchzip, ... }:
+let
+  desktopItem = makeDesktopItem {
+    name = "RCPTT";
+    exec = "rcptt";
+    icon = "rcptt";
+    comment = "RCP Testing Tool is a project for GUI testing automation of Eclipse-based applications. RCPTT is fully aware about Eclipse Platform's internals, hiding this complexity from end users and allowing QA engineers to create highly reliable UI tests at great pace.";
+    desktopName = "RCPTT";
+    genericName = "RCP Testing Tool";
+    categories = [ "Development" "IDE" "Java"  ];
+  };
```

Comment:
There's no need to put this in a `let` block. Please declare this inside `mkDerivation`.